src/graph/strongly-connected-components.md

```diff
@@ -18,7 +18,7 @@ The most important property of the condensation graph is that it is **acyclic**.
 The algorithm described in the next section extracts all strongly connected components in a given graph. It is quite easy to build a condensation graph then.
 
 ## Description of the algorithm
-Described algorithm was independently suggested by Kosaraju and Sharir at 1979. This is an easy-to-implement algorithm based on two series of [depth first search](http://e-maxx-eng.appspot.com/graph/depth-first-search.html), and working for $O(n + m)$ time.
+Described algorithm was independently suggested by Kosaraju and Sharir at 1979. This is an easy-to-implement algorithm based on two series of [depth first search](https://cp-algorithms.com/graph/depth-first-search.html), and working for $O(n + m)$ time.
```

Comment:
For this link and the next one, it would be better to use relative links instead of the absolute ones: [depth first search](./graph/depth-first-search.html)